components/Chat/Message/MessageReactions/MessageReactions.types.ts

```diff
@@ -38,5 +38,5 @@ export type ReactorDetails = {
   address: string;
   userName?: string;
   avatar?: string;
-  reactionTime: number;
+  // reactionTime: number; // TODO
```

Comment:
Is this still todo?

Reply:
No you're right removed it. At least for now it seems we're not doing anything with showing reaction timestamp